encoding/basics-and-workflow.md

```diff
@@ -29,7 +29,144 @@ provided by VSEdit.
-examples for qaac, flac
+As said earlier, only qaac will require configuration,
+and everythig else can simply be extracted wherever you like.
+But, for the ease of use, we suggest adding them to PATH,
+and for the purpose of this guide we will assume you've done it.
+
+
+### Decoding audio with ffmpeg and piping it out
+
+Basic ffmpeg usage is very simple—you just need to specify the input file,
+and the name and extension of your desired output file, like this:
+```
+ffmpeg -i "input.dts" "output.wav"
+```
+This command will decode the DTS audio file and save it as a WAV file.
+Of course, this command doesn't contain any configuration whatsoever,
+so ffmpeg will resort to using its defaults.
+
+For transcoding audio,
+only a little bit of configuration will be needed:
+```
+ffmpeg -i "input.dts" -acodec pcm_s24le "output.wav"
+```
+The `-acodec pcm_s24le` parameter will make sure
+that ffmpeg will output in 24 bits;
+otherwise it will output in 16 bits by default.
+If your source file is 16 bits, change this parameter to `pcm_s16le`,
+or simply skip it.
+
+The command above will decode the source file
+and save a resulting WAV file on your drive.
+You can then use this WAV file to encode a FLAC or AAC file,
+but there is a faster and more convinient way to do that—piping.
+Piping skips the process of writing
+and reading the data from the drive
+and simply sends the data straight from one program to another.
+
+To pipe from ffmpeg, specify the output format as WAV using the `-f` option,
+replace the output filename with a hyphen and place a pipe symbol at the end,
+which will be used to separate the ffmpeg command from your encoder command,
+like this:
+```
+ffmpeg -i "input.dts" -acodec pcm_s24le -f wav - |
+```
+
+
+### Encoding with FLAC
+
+To encode with FLAC, use the following command:
+```
+flac -8 --ignore-chunk-sizes "input.wav" -o "output.flac"
+```
+`-8` sets the encoding level to 8—the highest possible.
+This will result in the best compression,
+but will be slightly slower than lower levels.
+FLAC encoding is fast, so just stick with level 8.  
+`--ignore-chunk-sizes` is needed
+because sometimes the WAV header doesn't contain
+any information about the length of the file,
+or that information is incorrect,
+which makes it stop the encoding before the file actually ends.
+With this option FLAC will ignore the header
+and just encode the file from the beginning to the end.
+
+To encode audio piped from ffmpeg,
+replace the input filename with a hyphen
+and place the whole command after the ffmpeg command,
+like this:
+```
+ffmpeg -i "input.dts" -acodec pcm_s24le -f wav -| flac -8 --ignore-chunk-sizes - -o "output.flac"
+```
+
+
+### Encoding with qaac
+
+First, set up qaac:
+* go to [qaac's download page][qaac] and download the newest build
+(at the time of writing, `qaac_2.68.zip`) and the `makeportable.zip` file
+* extract the `x64` folder wherever you want your qaac folder to be,
+then extract contents of `makeportable.zip` to it
```

Comment:
This should be another step, shouldn't it.

Also, this should in general be an ordered list. Use `1.` for every item.